src/cohortMembers/cohortMembers.controller.ts

```diff
@@ -179,4 +180,29 @@ export class CohortMembersController {
       .buildCohortMembersAdapter()
       .deleteCohortMemberById(tenantid, cohortMembershipId, response);
   }
+
+  @UseFilters(new AllExceptionsFilter(APIID.COHORT_MEMBER_CREATE))
+  @Post("/bulkCohortMembers")
```

Comment:
@Shubham4026 the endpoint /user/v1/cohortmember/bulkCohortMembers wont look good. Please change it to
/user/v1/cohortmember/bulkCreate

Reply:
Done Please check and Merge